src/components/Comment.jsx

```diff
@@ -159,6 +159,10 @@ class Comment extends React.Component {
         return <div onClick={() => this.safeSetState({editMode: true})} className={'mentions-edit fa fa-edit'}/>;
     }
 
+    renderSuggestion(entry, search, highlightedDisplay, index) {
+        return <span>{highlightedDisplay} <i><small>{entry.email}</small></i></span>;
```

Comment:
Isn't something like that already defined somewhere else in the project ?

Reply:
Well, I took the point of view that they are different components and they could work differently. 
I will make it more generic with a new CoamMention component.

Reply:
Can you come up with potential (even hypothetical) example where the suggestions in the add new comment box would be different from the one while editing comment and we wouldn't end up having someone reporting it as a bug ?

Reply:
Well so far they work differently as one input has autofocus, different height.. but I have been more precise and create a single file with a renderer that can be used by both.